salt/grains/onyx.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+'''
+Grains for Onyx OS Switches Proxy minions
+
+.. versionadded: 2018.3.2 (Oxygen)
```

Comment:
This should actually be `Neon`. That is the current code name for the develop branch.